modules/internal/CPtr.chpl

```diff
@@ -377,6 +377,31 @@ module CPtr {
     return sizeof(x);
   }
 
+  /*
+    Return the offset of elements in a record.
```

Comment:
I suggest phrasing it as a singular "of a field" instead of as a plural "of elements".